Team-Charter.md

```diff
@@ -0,0 +1,132 @@
+
+However, just like standard team members, this does not mean that core team members are expected to manage conflicts or communication issues on their own. If they need assistance with crafting responses to sensitive or difficult situations, the team is here to support you.
+
+#### #3. Be a reliable team player
+
+In addition, there are two additional traits that are critical to being a core team member: **consistency** and **collaboration. **Joining the core team is a commitment to not only the public but your team as well.
+
+While the amount of time that each core team member can dedicate varies (which contributes to their consistency), it's also important to remember that our team is one that requires collaboration and teamwork.
+
+### Privileges
+
+On top of standard team member privileges:
+
+- Entry on the core team page on vuejs.org
+- Ability to use the title "Vue.js core team member" in public
+
+### Nomination and Approval
+
+- A core team member can be nominated by an existing core team member or the project lead.
+- Only current active team members can receive core team member nominations.
+- The nomination can only be approved by the project lead, after consulting the general consensus of the current core team members.
+
+## What if I am unable to meet these expectations?
+
+While a consistent presence is expected of team members, sometimes a team member needs to step away from the project temporarily or wants to move away from the project entirely. In these cases, some options team members have include:
+
+### 1. Taking some time off
+
+The benefit of being on a team is that you are not solely responsible for the entire project. Members are welcome to take time off of the project. We will remove you from the team page during your absence so that it’s clear who is still available on the team. You can still be a contributor during this time.
+
+### 2. Switching to an emeritus role
+
+The emeritus role is a status that allows team members to transition from their position on the team while still being recognized for their past contributions.
+
+There are two ways that team members can be switched into emeritus status:
+
+1. A team member voluntarily opts-in
+2. A consensus is reached by the core team and/or project lead to move a team member to move them into emeritus status if they are inactive for an extended period of time
+
+This is not a permanent status. They can transition back into an active team role if they are able to take on those responsibilities again and have the approval of the team.
+
+## Final Thoughts
+
+Thank you for taking the time to read this!
```

Comment:
This used to say who to reach out to. Do we want to keep that? Perhaps with the team email?